src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -319,6 +336,14 @@ function RemoteFunctions(experimental) {
             if (this.trigger) {
                 _trigger(element, "highlight", 1);
             }
+            
+            if (!window.event && !isInViewport(element)) {
```

Comment:
Question: not sure why the `window.event` check is needed here? Something to do with legacy IE's?